README.md

```diff
@@ -51,9 +51,6 @@ This plugin is compatible with:
 - [Composer][composer] **1.x** and **2.x** (Support for Composer v2 is available since [`v0.7.0`][v0.7])
 - [PHP_CodeSniffer][codesniffer] **2.x** and **3.x** (Support for PHP_CodeSniffer v3 is available since [`v0.4.0`][v0.4])
 
-
-> **ℹ️ Please Note:** [Composer treats _minor_ releases below 1.0.0 as _major_ releases][composer-manual-caret]. So version `0.7.x` (or higher) of this plugin must be _explicitly_ set as version constraint when using Composer 2.x or PHP 8.0. In other words: using `^0.6` will **not** work with Composer 2.x or PHP 8.0.
```

Comment:
Let's also remove the corresponding link on (old) line 263.

Reply:
Good catch. Thanks.

I propose instead to add the link to the text in the blockquote in the "Requiring the plugin from within your coding standard" section.